src/kube-runtime/src/init

```diff
@@ -78,6 +79,10 @@ PAI_RUNTIME_DIR=${PAI_WORK_DIR}/runtime.d
 
 PAI_LOG_DIR=${PAI_WORK_DIR}/logs/${FC_POD_UID}
 
```

Comment:
We may need to keep all history and logs for tracking. So to achieve Idempotent, can you use a different folder each time?

Reply:
Will move old logs to ${PAI_LOG_DIR}/pre_{index}

---
In reply to: [332809405](https://github.com/microsoft/pai/pull/3695#discussion_r332809405) [](ancestors = 332809405)